<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="2">

  <testsuite name="006.search"
    errors="0" failures="1" hostname="" id="" package="006.search" skipped="1"
    tests="2" time="0.9590" timestamp="Mon, 23 May 2016 03:45:03 GMT">
  
    <testcase name="search - no result" time="0.9590" assertions="2">

      <failure message="Testing if element &lt;#nav&gt; is present.">    at Object.exports.(anonymous function) [as search - no result] (/home/zxd/dev0/jadepress-theme-pi/test/e2e/specs/006.search.js:31:12)
    at Module.call (/home/zxd/dev0/jadepress-theme-pi/node_modules/nightwatch/lib/runner/module.js:63:34)
    at /home/zxd/dev0/jadepress-theme-pi/node_modules/nightwatch/lib/runner/testcase.js:97:29
    at _fulfilled (/home/zxd/dev0/jadepress-theme-pi/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/zxd/dev0/jadepress-theme-pi/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/zxd/dev0/jadepress-theme-pi/node_modules/q/q.js:796:13)
    at /home/zxd/dev0/jadepress-theme-pi/node_modules/q/q.js:556:49
    at runSingle (/home/zxd/dev0/jadepress-theme-pi/node_modules/q/q.js:137:13)
    at flush (/home/zxd/dev0/jadepress-theme-pi/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)</failure>

    
    </testcase>
  
  
    
    <testcase
      name="search - has result">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
